src/OodDesign/BookShelf.java

```diff
@@ -0,0 +1,117 @@
+//package OodDesign;
```

Comment:
Delete commented code, or put them in your EverNote, or uncomment them and add proper comments explanations